akka-remote/src/main/mima-filters/2.5.x.backwards.excludes/_existing.excludes

```diff
@@ -74,3 +74,10 @@ ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTra
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTransport.createFlightRecorderEventSink")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTransport.createFlightRecorderEventSink$default$1")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTransport.flightRecorder")
+
+# Remove secure-cookie
```

Comment:
Better to put it in its own file now of course ;)

```suggestion
# Remove secure-cookie #27809
```

Reply:
thanks, yeah I was lazy since rebase handled the file rename automatically